modules/core/client/views/issue.client.view.html

```diff
@@ -57,19 +57,33 @@
 					</md-button>
 				</md-fab-trigger>
 
-				<md-fab-actions>
-					<md-button aria-label="Share to Facebook" class="md-icon-button">
-						<md-icon md-svg-src="/modules/core/client/img/facebook.svg"></md-icon>
-						<md-tooltip>Facebook</md-tooltip>
-					</md-button>
+			<md-fab-actions>
+				<md-button aria-label="Share to Facebook" class="md-icon-button" ng-click="vm.share('facebook')">
+          <md-icon md-svg-src="/modules/core/client/img/facebook.svg"></md-icon>
+          <md-tooltip>Facebook</md-tooltip>
+        </md-button>
 
-					<md-button aria-label="Share on Twitter" class="md-icon-button">
-						<md-icon md-svg-src="/modules/core/client/img/twitter.svg"></md-icon>
-						<md-tooltip>Twitter</md-tooltip>
-					</md-button>
-				</md-fab-actions>
-			</md-fab-speed-dial>
-		</div>
+        <md-button aria-label="Share on Twitter" class="md-icon-button" ng-click="vm.share('twitter')">
+          <md-icon md-svg-src="/modules/core/client/img/twitter.svg"></md-icon>
+          <md-tooltip>Twitter</md-tooltip>
+        </md-button>
+
+        <md-button aria-label="Share on Google+" class="md-icon-button" ng-click="vm.share('google_plus')">
+          <md-icon md-svg-src="/modules/core/client/img/google-plus.svg"></md-icon>
+          <md-tooltip>Google+</md-tooltip>
+        </md-button>
+
+        <md-button aria-label="Share on LinkedIn" class="md-icon-button" ng-click="vm.share('linkedin')">
+          <md-icon md-svg-src="/modules/core/client/img/linkedin.svg"></md-icon>
+          <md-tooltip>LinkedIn</md-tooltip>
+        </md-button>
+
+        <md-button aria-label="Share on Reddit" class="md-icon-button" ng-click="vm.share('reddit')">
+          <md-icon md-svg-src="/modules/core/client/img/reddit.svg"></md-icon>
+          <md-tooltip>Reddit</md-tooltip>
+        </md-button>
+			</md-fab-actions>
+		</md-fab-speed-dial>
 	</div>
```

Comment:
</div> is missing from here

Reply:
I have added back the ```</div>```. From now on I will check all the views.